templates/commands/render/render.go

```diff
@@ -586,6 +600,24 @@ func executeOneStep(ctx context.Context, stepIdx int, step *spec.Step, sp *stepP
 	}
 }
 
+func loadInputFile(ctx context.Context, fs common.FS, path string) (map[string]string, error) {
+	f, err := fs.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("error opening input file: ReadFile(): %w", err)
+	}
+	defer f.Close()
+
+	m := make(map[string]string)
+	data, err := io.ReadAll(f)
```

Comment:
I think this could be simplified a bit using https://pkg.go.dev/os#ReadFile

Reply:
Done